### PR TITLE
4216: Update score enums.

### PR DESCRIPTION
**Pull request by raman-bt**

### CODE REVIEW

**Comment by APriestman** [comment]
Please fix conflicts

File: bindings/java/src/org/sleuthkit/datamodel/ScoringManager.java [hunk 1]
```diff
@@ -248,44 +247,45 @@ private long getContentCount(long dataSourceObjectId, Score aggregateScore, Case
...
-			return getContent(dataSourceObjectId, aggregateScore, connection);
+			return getContent(dataSourceObjectId, significance, connection);
 		} 
 	}
 
 	/**
-	 * Gets the contents with the specified score. Uses the specified transaction
-	 * to obtain the database connection.
+	 * Gets the contents with the specified score. Uses the specified
+	 * transaction to obtain the database connection.
```
Comment (APriestman) [inline bindings/java/src/org/sleuthkit/datamodel/ScoringManager.java:274]: It's passed a connection not a transaction

File: bindings/java/src/org/sleuthkit/datamodel/ScoringManager.java [hunk 2]
```diff
@@ -205,38 +205,37 @@ Score updateAggregateScore(long objId, Long dataSourceObjectId, Score resultScor
...
+			return getContentCount(dataSourceObjectId, significance, connection);
 		} 
 	}
 
 
 	/**
-	 * Get the count of contents with the specified score. Uses the specified
-	 * transaction to obtain the database connection.
+	 * Get the count of contents with the specified significance. Uses the
+	 * specified transaction to obtain the database connection.
```
Comment (APriestman) [inline bindings/java/src/org/sleuthkit/datamodel/ScoringManager.java:225]: It's passed a connection not a transaction. @param tag also says transaction.

**Review by APriestman — APPROVED** [review]

**Comment by raman-bt** [comment]
@rcordovano: could you merge this please?

**Pull request merged**